src/database/migration.rs

```diff
@@ -48,7 +48,7 @@ impl DatabaseMigrator {
     }
 
     // no database -> v1
-    fn to_one_from_none(conn: &Connection) -> Result<(), Error> {
+    fn upgrade_to_one_from_none(conn: &Connection) -> Result<(), Error> {
```

Comment:
this name was chosen for composability (i.e. `to_two_from_one(to_one_from_none(...))`), with the added upgrade this is slightly less readable `upgrade_to_two_from_one(upgrade_to_one_from_none(...))`, but on its own it might be better

Reply:
<img width="685" alt="screen shot 2018-04-11 at 3 34 16 pm" src="https://user-images.githubusercontent.com/1786772/38620545-f738f69a-3d9e-11e8-80ed-a230620f7be1.png">

Because of [that](https://github.com/rust-lang-nursery/rust-clippy/wiki#wrong_self_convention) warning, changed the signature but if you have something better in mind, let me know :)